qpdf: 11.10.0 -> 12.1.0

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from https://github.com/qpdf/qpdf/releases.


meta.description for qpdf is: C++ library and set of programs that inspect and manipulate the structure of PDF files

meta.homepage for qpdf is: https://qpdf.sourceforge.io/

meta.changelog for qpdf is: https://github.com/qpdf/qpdf/blob/v12.0.0/ChangeLog


###### Updates performed
- Version update

###### To inspect upstream changes

- [Release on GitHub](https://github.com/qpdf/qpdf/releases/tag/v12.0.0)

- [Compare changes on GitHub](https://github.com/qpdf/qpdf/compare/v11.10.0...v12.0.0)

###### Impact

<b>Checks done</b>

---

- built on NixOS

> [!CAUTION]
> A test defined in `passthru.tests` did not pass.
- found 12.0.0 with grep in /nix/store/lc1mkv6zk8flxv20358s28rh3nbrplcb-qpdf-12.0.0-bin
- found 12.0.0 in filename of file in /nix/store/lc1mkv6zk8flxv20358s28rh3nbrplcb-qpdf-12.0.0-bin

---

<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
2575 total rebuild path(s)

2575 package rebuild(s)

First fifty rebuilds by attrpath

activitywatch
apache-airflow
apacheHttpdPackages.mod_tile
apacheHttpdPackages_2_4.mod_tile
audiness
authentik
aw-watcher-afk
azure-cli-extensions.azure-iot
baserow
beets
beets-unstable
beetsPackages.beets
beetsPackages.beets-stable
beetsPackages.beets-unstable
bitwarden-menu
blockstream-electrs
borgbackup
borgmatic
browsr
buildbot
buildbot-full
buildbot-plugins.badges
buildbot-ui
buildbotPackages.buildbot
buildbotPackages.buildbot-full
buildbotPackages.buildbot-plugins.badges
buildbotPackages.buildbot-ui
buku
cairosvg
calibre
calibre-web
cantoolz
cfripper
checkov
cloudcompare
cmake-format
cmake-language-server
codechecker
conan
crackql
cups-browsed
cups-filters
datasette
datatrove
dbx
deja-dup
docling
duplicity
duply
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from the cache**:
```
nix-store -r /nix/store/lc1mkv6zk8flxv20358s28rh3nbrplcb-qpdf-12.0.0-bin \
  --option binary-caches 'https://cache.nixos.org/ https://nixpkgs-update-cache.nix-community.org/' \
  --option trusted-public-keys '
  nixpkgs-update-cache.nix-community.org-1:U8d6wiQecHUPJFSqHN9GSSmNkmdiFW7GW7WNAnHW0SM=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The nixpkgs-update cache is only trusted for this store-path realization.)
For the cached download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A qpdf https://github.com/r-ryantm/nixpkgs/archive/43ee432dcea551d1f47e4c20231b0f9f8d6aa12b.tar.gz
```
Or:
```
nix build github:r-ryantm/nixpkgs/43ee432dcea551d1f47e4c20231b0f9f8d6aa12b#qpdf
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/lc1mkv6zk8flxv20358s28rh3nbrplcb-qpdf-12.0.0-bin
ls -la /nix/store/lc1mkv6zk8flxv20358s28rh3nbrplcb-qpdf-12.0.0-bin/bin
```

---

</details>
<br/>



### Pre-merge build results

NixPkgs review skipped

---

###### Maintainer pings

cc @abbradar for [testing](https://github.com/ryantm/nixpkgs-update/blob/main/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

> [!TIP]
> As a maintainer, if your package is located under `pkgs/by-name/*`, you can comment **`@NixOS/nixpkgs-merge-bot merge`** to automatically merge this update using the [`nixpkgs-merge-bot`](https://github.com/NixOS/nixpkgs-merge-bot).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

MarcFontaine committed:
libacars: init at 2.2.0

pks-t committed:
opensmtpd.table-sqlite: init at 1.0.1

The opensmtpd-extra package has been deprecated in favor of standalone
repositories for each of the filters. Introduce table-sqlite, which is
one of these new standalone filters to read login info from an sqlite
database.

pks-t committed:
nixos/opensmtpd: adapt to changed filter location

The location of filters for OpenSMTPD has been changed to
"libexec/smtpd". Adapt the service accordingly.

pks-t committed:
opensmtpd.table-ldap: init at 1.0

The opensmtpd-extra package has been deprecated in favor of standalone
repositories for each of the filters. Introduce table-ldap, which is one
of these new standalone filters to read login info from an LDAP server.

pks-t committed:
opensmtpd.table-mysql: init at 1.2.1

The opensmtpd-extra package has been deprecated in favor of standalone
repositories for each of the filters. Introduce table-mysql, which is
one of these new standalone filters to read login info from a MySQL or
MariaDB database.

pks-t committed:
opensmtpd.table-passwd: init at 1.0.2

The opensmtpd-extra package has been deprecated in favor of standalone
repositories for each of the filters. Introduce table-passwd, which is
one of these new standalone filters to read login info from "passwd"
files.

pks-t committed:
opensmtpd.table-postgres: init at 1.1.1

The opensmtpd-extra package has been deprecated in favor of standalone
repositories for each of the filters. Introduce table-postgres, which
is one of these new standalone filters to read login info from a
Postgres database.

pks-t committed:
opensmtpd.table-redis: init at 1.0.1

The opensmtpd-extra package has been deprecated in favor of standalone
repositories for each of the filters. Introduce table-redis, which is
one of these new standalone filters to read login info from a Redis
instance.

Building table-redis does not work out of the box: while the project
correctly includes the hiredis include directory, the header itself is
contained in a "hiredis/" subdirectory. Add a patch to fix this.

pks-t committed:
opensmtpd.table-socketmap: init at 1.1.1

The opensmtpd-extra package has been deprecated in favor of standalone
repositories for each of the filters. Introduce table-socketmap, which
is one of these new standalone filters to read login info from a Unix
socket.

ferrine committed:
home-assistant-custom-components.yandex-station: init at 3.18.3

pks-t committed:
opensmtpd-extras: drop in favor of standalone tables

Upstream has archived the OpenSMTPD-extras repository with 82c49af (add
deprecation notice, 2024-10-22). Instead, it has created a set of
repositories, each containing one of the tables. The old tables aren't
compatible anymore with recent OpenSMTPD versions due to changed
protocols. Other extras, like the queues, don't have an alternative to
the best of my knowledge.

Furthermore, opensmtpd-extras doesn't even build anymore, furthermore
indicating that there aren't any users of this package. Remove the
"opensmtpd-extras" package and raise errors pointing to the new
standalone tables.

Add a release note.

wegank reviewed (changes requested):
Breaks `pdfmixtool`.

r-ryantm committed:
python312Packages.skrl: 1.4.2 -> 1.4.3

LordMZTE committed:
ponyc: 0.58.6 -> 0.59.0

This also fixes a build issue that broke the package before, further
described in #392168

Resolves #392168

r-ryantm committed:
qpdf: 11.10.0 -> 12.1.0

r-ryantm commented:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from https://github.com/qpdf/qpdf/releases.


meta.description for qpdf is: C++ library and set of programs that inspect and manipulate the structure of PDF files

meta.homepage for qpdf is: https://qpdf.sourceforge.io/

meta.changelog for qpdf is: https://github.com/qpdf/qpdf/blob/v12.1.0/ChangeLog


###### Updates performed
- Version update

###### To inspect upstream changes

- [Release on GitHub](https://github.com/qpdf/qpdf/releases/tag/v12.1.0)

- [Compare changes on GitHub](https://github.com/qpdf/qpdf/compare/v11.10.0...v12.1.0)

###### Impact

<b>Checks done</b>

---

- built on NixOS

> [!CAUTION]
> A test defined in `passthru.tests` did not pass.
- found 12.1.0 with grep in /nix/store/c21p97x0iliwlp2hgdybxpam91y08ddi-qpdf-12.1.0-bin
- found 12.1.0 in filename of file in /nix/store/c21p97x0iliwlp2hgdybxpam91y08ddi-qpdf-12.1.0-bin

---

<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
2613 total rebuild path(s)

2613 package rebuild(s)

First fifty rebuilds by attrpath

activitywatch
aider-chat-full
aider-chat-with-help
apache-airflow
apacheHttpdPackages.mod_tile
apacheHttpdPackages_2_4.mod_tile
audiness
authentik
aw-watcher-afk
azure-cli-extensions.azure-iot
baserow
beets
beets-unstable
beetsPackages.beets
beetsPackages.beets-stable
beetsPackages.beets-unstable
bitwarden-menu
blockstream-electrs
borgbackup
borgmatic
browsr
buildbot
buildbot-full
buildbot-plugins.badges
buildbot-ui
buildbotPackages.buildbot
buildbotPackages.buildbot-full
buildbotPackages.buildbot-plugins.badges
buildbotPackages.buildbot-ui
buku
cairosvg
calibre
calibre-web
cantoolz
cfripper
checkov
cloudcompare
cmake-format
cmake-language-server
codechecker
conan
crackql
cups-browsed
cups-filters
datasette
datatrove
dbx
deja-dup
docling
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from the cache**:
```
nix-store -r /nix/store/c21p97x0iliwlp2hgdybxpam91y08ddi-qpdf-12.1.0-bin \
  --option binary-caches 'https://cache.nixos.org/ https://nixpkgs-update-cache.nix-community.org/' \
  --option trusted-public-keys '
  nixpkgs-update-cache.nix-community.org-1:U8d6wiQecHUPJFSqHN9GSSmNkmdiFW7GW7WNAnHW0SM=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The nixpkgs-update cache is only trusted for this store-path realization.)
For the cached download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A qpdf https://github.com/r-ryantm/nixpkgs/archive/2c4db131dc4f6de294abd3108b856aa077edee22.tar.gz
```
Or:
```
nix build github:r-ryantm/nixpkgs/2c4db131dc4f6de294abd3108b856aa077edee22#qpdf
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/c21p97x0iliwlp2hgdybxpam91y08ddi-qpdf-12.1.0-bin
ls -la /nix/store/c21p97x0iliwlp2hgdybxpam91y08ddi-qpdf-12.1.0-bin/bin
```

---

</details>
<br/>



### Pre-merge build results

NixPkgs review skipped

---

###### Maintainer pings

cc @abbradar for [testing](https://github.com/ryantm/nixpkgs-update/blob/main/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

> [!TIP]
> As a maintainer, if your package is located under `pkgs/by-name/*`, you can comment **`@NixOS/nixpkgs-merge-bot merge`** to automatically merge this update using the [`nixpkgs-merge-bot`](https://github.com/NixOS/nixpkgs-merge-bot).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

gauravghodinde committed:
rexi: init at 1.2.0

gauravghodinde committed:
maintainers: add gauravghodinde

Update pkgs/by-name/re/rexi/package.nix

Co-authored-by: Aleksana <alexander.huang.y@gmail.com>

tomjnixon committed:
maintainers: add tomjnixon

tomjnixon committed:
python3Packages.opencamlib: init at 2023.01.11

tomjnixon committed:
freecad: use opencamlib

This allows use of some experimental CAM features, which must be
explicitly enabled:

https://wiki.freecad.org/CAM_experimental

khaneliman committed:
buildMozillaMach: apple-sdk_14 -> apple-sdk_15 starting with version 138

0:01.18 ERROR: SDK version "14.4" is too old. Please upgrade to at
least 15.2. Try updating your system Xcode. Newer versions of firefox
require apple-sdk_15.

Commit:
conan: 2.9.1 -> 2.15.1

Diff: https://github.com/conan-io/conan/compare/refs/tags/2.9.1...refs/tags/2.15.1

Changelog: https://github.com/conan-io/conan/releases/tag/2.15.1

Commit:
conduktor: 2.15.1 -> 2.24.9

Diff: https://github.com/conduktor/builds/compare/v2.15.1...v2.24.9

Changelog: https://www.conduktor.io/changelog/#2.24.9

Commit:
contour: 0.5.1.7247 -> 0.6.1.7494

Diff: https://github.com/contour-terminal/contour/compare/v0.5.1.7247...v0.6.1.7494

Changelog: https://github.com/contour-terminal/contour/raw/v25.05pre-git/Changelog.md

Commit:
CuboCore.coreinfo: 4.5.0 -> 5.0.0

Commit:
CuboCore.corekeyboard: 4.5.0 -> 5.0.0

Diff: https://gitlab.com/cubocore/coreapps/corekeyboard/-/compare/v4.5.0...v5.0.0

Commit:
CuboCore.corepad: 4.5.0 -> 5.0.0

Diff: https://gitlab.com/cubocore/coreapps/corepad/-/compare/v4.5.0...v5.0.0

Commit:
CuboCore.corepaint: 4.5.0 -> 5.0.0

Diff: https://gitlab.com/cubocore/coreapps/corepaint/-/compare/v4.5.0...v5.0.0

Commit:
CuboCore.corepdf: 4.5.0 -> 5.0.0

Diff: https://gitlab.com/cubocore/coreapps/corepdf/-/compare/v4.5.0...v5.0.0

Commit:
qdocumentview: init at 0.3.0

Commit:
CuboCore.corepins: 4.5.0 -> 5.0.0

Diff: https://gitlab.com/cubocore/coreapps/corepins/-/compare/v4.5.0...v5.0.0

Commit:
CuboCore.corerenamer: 4.5.0 -> 5.0.0

Diff: https://gitlab.com/cubocore/coreapps/corerenamer/-/compare/v4.5.0...v5.0.0

Commit:
CuboCore.coreshot: 4.5.0 -> 5.0.0

Diff: https://gitlab.com/cubocore/coreapps/coreshot/-/compare/v4.5.0...v5.0.0

Commit:
CuboCore.corestats: 4.5.0 -> 5.0.0

Commit:
CuboCore.corestuff: 4.5.0 -> 5.0.0

Diff: https://gitlab.com/cubocore/coreapps/corestuff/-/compare/v4.5.0...v5.0.0

Commit:
CuboCore.coreterminal: 4.5.0 -> 5.0.0

Diff: https://gitlab.com/cubocore/coreapps/coreterminal/-/compare/v4.5.0...v5.0.0

Commit:
CuboCore.coretime: 4.5.0 -> 5.0.1

Diff: https://gitlab.com/cubocore/coreapps/coretime/-/compare/v4.5.0...v5.0.1

Commit:
CuboCore.coretoppings: 4.5.0 -> 5.0.0

Diff: https://gitlab.com/cubocore/coreapps/coretoppings/-/compare/v4.5.0...v5.0.0

Commit:
CuboCore.coreuniverse: 4.5.0 -> 5.0.0

Diff: https://gitlab.com/cubocore/coreapps/coreuniverse/-/compare/v4.5.0...v5.0.0

Commit:
crc: 2.36.0 -> 2.49.0

Commit:
croaring: 2.0.3 -> 4.3.1

Diff: https://github.com/RoaringBitmap/CRoaring/compare/v2.0.3...v4.3.1

Commit:
csvkit: 2.0.1 -> 2.1.0

Changelog: https://github.com/wireservice/csvkit/blob/2.1.0/CHANGELOG.rst

Commit:
python313Packages.agate: 1.11.0 -> 1.13.0

Diff: https://github.com/wireservice/agate/compare/refs/tags/1.11.0...refs/tags/1.13.0

Changelog: https://github.com/wireservice/agate/blob/1.13.0/CHANGELOG.rst

Commit:
bitcomet: 2.12.1 -> 2.13.1

Commit:
saber: 0.25.4 -> 0.25.5

Commit:
simple-live-app: 1.7.6 -> 1.7.7

Commit:
proxypin: add update script

Commit:
proxypin: 1.1.7 -> 1.1.8

Commit:
chatmcp: 0.0.21-alpha -> 0.0.26-alpha

Commit:
cups-bjnp: 1.2.2 -> 2.0.3

Commit:
dbgate: 6.2.0 -> 6.3.3

Commit:
ddosify: 1.0.6 -> 2.6.0

Commit:
detox: 1.4.5 -> 2.0.0

Diff: https://github.com/dharple/detox/compare/v1.4.5...v2.0.0

Changelog: https://github.com/dharple/detox/blob/v2.0.0/CHANGELOG.md

Commit:
diylc: 4.18.0 -> 5.2.0

Commit:
dnsviz: 0.9.4 -> 0.11.1

Diff: https://github.com/dnsviz/dnsviz/compare/v0.9.4...v0.11.1

Commit:
doitlive: 5.0.0 -> 5.1.0

Changelog: https://github.com/sloria/doitlive/blob/5.1.0/CHANGELOG.rst

Commit:
dpdk: 24.07 -> 25.03

Commit:
butterfly: 2.2.4 -> 2.3.0

Diff: https://github.com/LinwoodDev/Butterfly/compare/refs/tags/v2.2.4...refs/tags/v2.3.0

Commit:
dtbloader: 1.2.2 -> 1.4.0

Diff: https://github.com/TravMurav/dtbloader/compare/refs/tags/1.2.2...refs/tags/1.4.0

Changelog: https://github.com/TravMurav/dtbloader/releases/tag/1.4.0

Commit:
dump1090: 9.0 -> 10.0.1

Diff: https://github.com/flightaware/dump1090/compare/v9.0...v10.0.1

Commit:
e2tools: 0.1.0 -> 0.1.2

Diff: https://github.com/e2tools/e2tools/compare/v0.1.0...v0.1.2

Commit:
forbidden: 12.6 -> 13.2

Diff: https://github.com/ivan-sincek/forbidden/compare/refs/tags/v12.6...refs/tags/v13.2

Changelog: https://github.com/ivan-sincek/forbidden/releases/tag/v13.2

Commit:
python313Packages.bot-safe-agents: init at 1.0

Commit:
fuc: 2.2.0 -> 3.0.1

Diff: https://github.com/SUPERCILEX/fuc/compare/2.2.0...3.0.1

r-ryantm committed:
python312Packages.symengine: 0.14.0 -> 0.14.1

Gliczy committed:
signal-desktop-bin: 7.47.0 -> 7.52.0

Gliczy committed:
signal-desktop-bin(aarch64-linux): 7.47.0 ->7.52.0

Gliczy committed:
signal-desktop-bin: replace apple emoji sheets
Signal now ships with `emoji-sheet-32.webp` and `emoji-sheet-64.webp`, both use Apple emoji.
This commit removes both and replace them with the equivalent noto-fonts emoji sheets.

r-ryantm committed:
borgmatic: 2.0.2 -> 2.0.4

r-ryantm committed:
testssl: 3.0.9 -> 3.2.0

winterqt committed:
jujutsu: fix tests in darwin sandbox

phaer committed:
age-plugin-se: build on linux as well

Commit:
gcompris: 4.3 -> 25.0

happysalada committed:
pythonPackages.polars: 1.21.0 -> 1.27.1

CherryKitten committed:
gotosocial: move to codeberg

doronbehar committed:
sshd-openpgp-auth: 0.3.0 -> 0.3.1

doronbehar committed:
ssh-openpgp-auth: 0.2.2 -> 0.2.3

r-ryantm committed:
python312Packages.securesystemslib: 1.2.0 -> 1.3.0

UlyssesZh committed:
chrome-pak-customizer: init at 2.0-unstable-2021-06-24

r-ryantm committed:
sitespeed-io: 37.3.2 -> 37.4.2

Gliczy committed:
signal-desktop-bin: cleanup `copy-noto-emoji.py`

Gliczy committed:
signal-desktop-bin(darwin): 7.51.0 -> 7.52.0

bobby285271 committed:
budgie-desktop: Explicitly set overlay-key to Super_L

Mutter 48 defaults to 'Super' for overlay-key, which is not supported in
magpie (note that we don't build magpie with with_shared_components). As
a result the budgie menu is not popping up with the "Windows" key.

The upstream author also experienced crashes like "The mapping function given
to g_settings_get_mapped() for key 'overlay-key' in schema 'org.gnome.mutter'
returned FALSE when given a NULL value" though I cannot reproduce that yet.

bobby285271 committed:
budgie-gsettings-overrides: Remove duplicated edge-tiling override

This is already overridden in 20_solus-project.budgie.wm.gschema.override,
which will be copied here.

r-ryantm committed:
ethercat: 1.6.3 -> 1.6.4

Commit:
cherry-studio: 1.2.4 -> 1.2.7

Commit:
gigedit: 1.1.1 -> 1.2.1

Commit:
linuxsampler: refactor

Commit:
gitify: 6.1.0 -> 6.3.0

Diff: https://github.com/gitify-app/gitify/compare/refs/tags/v6.1.0...refs/tags/v6.3.0

Changelog: https://github.com/gitify-app/gitify/releases/tag/v6.3.0

pks-t committed:
opensmtpd-filter-rspamd: migrate to by-name

pks-t committed:
opensmtpd: migrate to by-name

Commit:
gkrellm: 2.3.11 -> 2.4.0

r-ryantm committed:
peergos: 1.1.0 -> 1.2.0

tetov committed:
luaPackages.cqueues: enable builds for lua 5.4

pbsds committed:
ldeep: unbreak by relaxing termcolor

fixes https://hydra.nixos.org/build/294996567

r-ryantm committed:
netbird-ui: 0.40.0 -> 0.43.0

r-ryantm committed:
fleetctl: 4.64.1 -> 4.67.1

numinit committed:
ponyc, pony-corral: move to by-name

r-ryantm committed:
your_spotify: 1.12.0 -> 1.13.1

r-ryantm committed:
python312Packages.yalexs-ble: 2.5.7 -> 2.6.0

r-ryantm committed:
python312Packages.python-hcl2: 7.1.0 -> 7.2.0

wegank committed:
python312Packages.skrl: drop support for python 3.6-3.7

Co-authored-by: Nick Cao <nickcao@nichi.co>

flokli committed:
secretscanner: build with vectorscan instead of hyperscan

See https://github.com/NixOS/nixpkgs/issues/310772 for context.

This brings us aarch64-linux support for free.

trofi committed:
bombono: bin to boost183 to fix the build

Without the change `bombono` build fails on `master` as
https://hydra.nixos.org/build/294719057:

    In file included from src/mbase/project/_extlibs_.h:61,
                     from src/mgui/_extlibs_.h:31,
                     from src/mgui/_pc_.h:29,
                     from src/mgui/dialog.cpp:22:
    src/mlib/sdk/bfs.h:12:10: fatal error: boost/filesystem/convenience.hpp: No such file or directory
       12 | #include <boost/filesystem/convenience.hpp> // fs::create_directories()
          |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

drupol committed:
starship: cleanup

r-ryantm committed:
nuclei-templates: 10.1.7 -> 10.2.0

r-ryantm committed:
python312Packages.pyghmi: 1.5.77 -> 1.6.0

petrzjunior committed:
paps: 0.7.1 -> 0.8.0, switch to meson

r-ryantm committed:
chirpstack-rest-api: 4.11.0 -> 4.12.0

r-ryantm committed:
chirpstack-concentratord: 4.4.7 -> 4.4.8

r-ryantm committed:
chirpstack-mqtt-forwarder: 4.3.1 -> 4.3.2

r-ryantm committed:
chirpstack-udp-forwarder: 4.1.8 -> 4.1.10

wegank committed:
python312Packages.skrl: add changelog

trofi committed:
xsokoban: fix build agaist gcc-14

It's a really old program. Let's disable most implicit
declarations to restore the build.

numinit committed:
androidenv: support linking "latest" version of all plugins

As noted in #379534, the NDK linking broke during refactoring. Add a test
for that, and support linking the latest for all platform plugins that
take at least one version.

r-ryantm committed:
roddhjav-apparmor-rules: 0-unstable-2025-04-16 -> 0-unstable-2025-04-25

numinit committed:
androidenv: update docs to explain the latest version

r-ryantm committed:
python312Packages.onedrive-personal-sdk: 0.0.13 -> 0.0.14

natsukium committed:
python312Packages.yalexs-ble: 2.5.7 -> 2.6.0 (#402325)

r-ryantm committed:
wxmaxima: 25.01.0 -> 25.04.0

r-ryantm committed:
rip2: 0.9.3 -> 0.9.4

natsukium committed:
python312Packages.securesystemslib: 1.2.0 -> 1.3.0 (#401401)

natsukium committed:
python312Packages.symengine: 0.14.0 -> 0.14.1 (#400814)

nixpkgs-merge-bot[bot] committed:
ethercat: 1.6.3 -> 1.6.4 (#401799)

Co-authored-by: stv0g <post@steffenvogel.de>

nixpkgs-merge-bot[bot] committed:
chirpstack-rest-api: 4.11.0 -> 4.12.0 (#402365)

Co-authored-by: stv0g <post@steffenvogel.de>

nixpkgs-merge-bot[bot] committed:
chirpstack-concentratord: 4.4.7 -> 4.4.8 (#402366)

Co-authored-by: stv0g <post@steffenvogel.de>

nixpkgs-merge-bot[bot] committed:
chirpstack-mqtt-forwarder: 4.3.1 -> 4.3.2 (#402367)

Co-authored-by: stv0g <post@steffenvogel.de>

nixpkgs-merge-bot[bot] committed:
chirpstack-udp-forwarder: 4.1.8 -> 4.1.10 (#402370)

Co-authored-by: stv0g <post@steffenvogel.de>

drupol committed:
starship: cleanup (#402349)

nixpkgs-merge-bot[bot] committed:
borgmatic: 2.0.2 -> 2.0.4 (#401040)

Co-authored-by: x123 <x123@users.noreply.github.com>

r-ryantm committed:
readarr: 0.4.14.2782 -> 0.4.15.2787

drupol committed:
gitify: 6.1.0 -> 6.3.0 (#401939)

drupol committed:
gcompris: 4.3 -> 25.0 (#401292)

drupol committed:
cherry-studio: 1.2.4 -> 1.2.7 (#401788)

drupol committed:
gigedit: 1.1.1 -> 1.2.1 (#401936)

drupol committed:
fuc: 2.2.0 -> 3.0.1 (#400698)

drupol committed:
butterfly: 2.2.4 -> 2.3.0 (#400635)

drupol committed:
dtbloader: 1.2.2 -> 1.4.0 (#400658)

drupol committed:
dpdk: 24.07 -> 25.03 (#400621)

drupol committed:
detox: 1.4.5 -> 2.0.0 (#400587)

drupol committed:
dbgate: 6.2.0 -> 6.3.3 (#400498)

drupol committed:
cups-bjnp: 1.2.2 -> 2.0.3 (#400475)

drupol committed:
dump1090: 9.0 -> 10.0.1 (#400660)

drupol committed:
e2tools: 0.1.0 -> 0.1.2 (#400664)

drupol committed:
bitcomet: 2.12.1 -> 2.13.1 (#400441)

drupol committed:
saber: 0.25.4 -> 0.25.5 (#400445)

drupol committed:
simple-live-app: 1.7.6 -> 1.7.7 (#400449)

drupol committed:
proxypin: 1.1.7 -> 1.1.8 (#400453)

drupol committed:
chatmcp: 0.0.21-alpha -> 0.0.26-alpha (#400454)

drupol committed:
diylc: 4.18.0 -> 5.2.0 (#400595)

drupol committed:
doitlive: 5.0.0 -> 5.1.0 (#400614)

drupol committed:
convertall: 0.8.0 -> 1.0.1 (#400204)

drupol committed:
CuboCore: 4.5.0 -> 5.0.0 (#400220)

drupol committed:
croaring: 2.0.3 -> 4.3.1 (#400341)

drupol committed:
csvkit: 2.0.1 -> 2.1.0; python313Packages.agate: 1.11.0 -> 1.13.0 (#400348)

drupol committed:
conan: 2.9.1 -> 2.15.1 (#400174)

drupol committed:
confluent-cli: 4.16.0 -> 4.25.0 (#400197)

r-ryantm committed:
syft: 1.22.0 -> 1.23.1

drupol committed:
libcef: refactor (#397452)

drupol committed:
flclash: 0.8.80 -> 0.8.82 (#397179)

drupol committed:
windsend: init at 1.5.1 (#396431)

drupol committed:
fvm: init at 3.2.1 (#396140)

drupol committed:
kazumi: 1.5.6 -> 1.6.5 (#394274)

drupol committed:
venera: 1.2.4 -> 1.4.0 (#388162)

drupol committed:
surface-control: 0.4.3-2 -> 0.4.7-1 (#386296)

drupol committed:
tiny-rdm: init at 1.2.3 (#398390)

drupol committed:
reqable: init at 2.33.8 (#398468)

drupol committed:
cromite: 134.0.6998.108 -> 135.0.7049.100 (#396081)

drupol committed:
alisthelper: init at 0.2.0-unstable-2025-01-04 (#396137)

drupol committed:
xmind: 8-update9 -> 25.01.01061-202501070800 (#393425)

drupol committed:
yo: remove package-lock.json (#397568)

wolfgangwalther committed:
sitespeed-io: 37.3.2 -> 37.4.2 (#398870)

r-ryantm committed:
python312Packages.conda-libmamba-solver: 25.3.0 -> 25.4.0

doronbehar committed:
ssh-openpgp-auth: 0.2.2 -> 0.2.3 (#401398)

7c6f434c committed:
xsokoban: fix build agaist gcc-14 (#402377)

drupol committed:
Revert "dpdk: 24.07 -> 25.03"

drupol committed:
Revert "dpdk: 24.07 -> 25.03" (#402456)

numinit committed:
ponyc: 0.58.6 -> 0.59.0 (#396553)

drupol committed:
rexi: init at 1.2.0 (#398469)

r-vdp committed:
age-plugin-se: build on linux (#401265)

drupol committed:
slint-viewer: init at 1.10.0 (#394944)

drupol committed:
hanken-grotesk: init at 0-unstable-2024-01-30 (#397215)

drupol committed:
chrome-pak-customizer: init at 2.0-unstable-2021-06-24 (#401482)

r-ryantm committed:
mycelium: 0.5.7 -> 0.6.0

GaetanLepage committed:
ocserv: add otp support (#395801)

Mic92 committed:
signal-desktop-bin(x86_64-linux; aarch64-linux; darwin): 7.47.0 -> 7.52.0; signal-desktop-bin: replace apple emoji sheets  (#400945)

FliegendeWurst committed:
libacars: init at 2.2.0 (#402348)

fabaff committed:
nuclei-templates: 10.1.7 -> 10.2.0 (#402355)

happysalada committed:
pythonPackages.polars: 1.21.0 -> 1.27.1 (#401298)

fabaff committed:
ldeep: unbreak by relaxing termcolor (#402068)

etu committed:
testssl: 3.0.9 -> 3.2.0 (#401112)

Commit:
reflection-cpp: init at 0.2.0

nixpkgs-merge-bot[bot] committed:
readarr: 0.4.14.2782 -> 0.4.15.2787 (#402434)

Co-authored-by: jocelynthode <jocelynthode@users.noreply.github.com>

fabaff committed:
forbidden: refactor

GaetanLepage committed:
python312Packages.batchspawner: fix

GaetanLepage committed:
python312Packages.jupyterhub: cleanup

r-vdp committed:
nixos/mycelium: pull in network-online.target

The test was failing because it waits for network-online.target which
was not part of the transaction.

fabaff committed:
forbidden: 12.6 -> 13.2 (#400696)

fabaff committed:
secretscanner: build with vectorscan instead of hyperscan (#402329)

GaetanLepage committed:
python312Packages.jupyterhub: 5.2.1 -> 5.3.0

Diff: https://github.com/jupyterhub/jupyterhub/compare/refs/tags/5.2.1...refs/tags/5.3.0

Changelog: https://github.com/jupyterhub/jupyterhub/blob/5.3.0/docs/source/reference/changelog.md

doronbehar committed:
wxmaxima: 25.01.0 -> 25.04.0 (#402409)

Commit:
dnsviz: move to pkgs/by-name

nixpkgs-merge-bot[bot] committed:
your_spotify: 1.12.0 -> 1.13.1 (#402213)

Co-authored-by: PatrickDaG <PatrickDaG@users.noreply.github.com>

katexochen committed:
libnvidia-container: 1.17.2 -> 1.17.6

Signed-off-by: Paul Meyer <katexochen0@gmail.com>

r-ryantm committed:
zed-editor: 0.183.10 -> 0.183.11

dotlambda committed:
python312Packages.onedrive-personal-sdk: 0.0.13 -> 0.0.14 (#402393)

teto committed:
yaziPlugins.rsync: init at 0-unstable-2025-04-12

K900 committed:
Revert "slint-viewer: init at 1.10.0"

K900 committed:
Revert "slint-viewer: init at 1.10.0" (#402486)

drupol committed:
gkrellm: 2.3.11 -> 2.4.0 (#401944)

katexochen committed:
libnvidia-container: renew

- use finalAttrs
- use tag in fetchFromGithub
- no meta = with lib;

Signed-off-by: Paul Meyer <katexochen0@gmail.com>

r-vdp committed:
mycelium: 0.5.7 -> 0.6.0 (#402466)

drupol committed:
crc: 2.36.0 -> 2.49.0 (#400339)

drupol committed:
contour: 0.5.1.7247 -> 0.6.1.7494 (#400200)

drupol committed:
ddosify: 1.0.6 -> 2.6.0 (#400560)

nixpkgs-merge-bot[bot] committed:
rip2: 0.9.3 -> 0.9.4 (#402411)

Co-authored-by: matthiasbeyer <mail@beyermatthias.de>

drupol committed:
conduktor: 2.15.1 -> 2.24.9 (#400188)

drupol committed:
dnsviz: 0.9.4 -> 0.11.1 (#400601)

natsukium committed:
python312Packages.jupyterhub: 5.2.1 -> 5.3.0, cleanup (#402473)

nixpkgs-merge-bot[bot] committed:
peergos: 1.1.0 -> 1.2.0 (#402030)

Co-authored-by: christoph-heiss <contact@christoph-heiss.at>

Defelo committed:
gotosocial: move to codeberg (#402238)

nikstur committed:
repart: Enable discard option (#397906)

natsukium committed:
python312Packages.pyghmi: 1.5.77 -> 1.6.0 (#402364)

wolfgangwalther committed:
opensmtpd-extras: drop in favor of standalone filters (#376913)

nixpkgs-merge-bot[bot] committed:
syft: 1.22.0 -> 1.23.1 (#402439)

Co-authored-by: 06kellyjac <06kellyjac@users.noreply.github.com>

Atemu committed:
androidenv: support linking "latest" version of all plugins (fixing NDK linking) (#402391)

GaetanLepage committed:
zed-editor: 0.183.10 -> 0.183.11 (#402484)

msanft committed:
libnvidia-container: 1.17.2 -> 1.17.6 (#402488)

drupol committed:
blender: fix openusd support (#399716)

NickCao committed:
opencamlib: init at 2023.01.11, and use in freecad (#400063)

r-ryantm committed:
spider: 2.36.73 -> 2.36.117

bobby285271 committed:
pantheon-tweaks: 2.2.0 -> 2.2.1

https://github.com/pantheon-tweaks/pantheon-tweaks/compare/2.2.0...2.2.1

Defelo committed:
netbird-ui: 0.40.0 -> 0.43.0 (#399601)

natsukium committed:
python312Packages.python-hcl2: 7.1.0 -> 7.2.0 (#402334)

natsukium committed:
python312Packages.conda-libmamba-solver: 25.3.0 -> 25.4.0 (#402447)

bobby285271 committed:
pantheon-tweaks: Explicitly add wingpanel-indicator-sound

This is for the max volume setting in MiscPane.

mweinelt committed:
buildMozillaMach: apple-sdk_14 -> apple-sdk_15 starting with version 138 (#400163)

r-ryantm committed:
bzip3: 1.5.1 -> 1.5.2

nixpkgs-merge-bot[bot] committed:
spider: 2.36.73 -> 2.36.117 (#402542)

Co-authored-by: j-mendez <jeffmendez19@gmail.com>

bobby285271 committed:
budgie-desktop: Explicitly set overlay-key to Super_L (#401792)

bobby285271 committed:
pantheon-tweaks: 2.2.0 -> 2.2.1 (#402544)

LeSuisse committed:
vorbis-tools: 1.4.2 -> 1.4.3 (#399564)

Defelo committed:
addr-book-combine: init at 0-unstable-2022-12-14 (#364890)

natsukium committed:
python312Packages.skrl: 1.4.2 -> 1.4.3 (#395738)

LeSuisse committed:
fleetctl: 4.64.1 -> 4.67.1 (#402172)

mweinelt committed:
home-assistant-custom-components.yandex-station: init at 3.18.3 (#390463)

dotlambda committed:
bzip3: 1.5.1 -> 1.5.2 (#402560)

matteo-pacini committed:
jujutsu: fix tests in darwin sandbox (#401211)

Sigmanificient committed:
paps: 0.7.1 -> 0.8.0, switch to meson (#402362)

dotlambda committed:
jellyfin-ffmpeg: 7.0.2-9 -> 7.1.1-1 (#399839)

vcunat committed:
nixos/kresd: add kresd-cli wrapper script (#388290)
